config/config.txt

```diff
@@ -83,13 +83,14 @@ LOG_ATTACK
 PROBABILITY EXTENDED 1
 PROBABILITY MALFUNCTION 1
 PROBABILITY MERCENARY 1
+PROBABILITY NINJA 1

```

Comment:
did Bay ever fix ninja before it was really crappy and just either overpowered or really weak depending on the version